lms/static/scripts/frontend_apps/utils/api.js

```diff
@@ -26,7 +27,8 @@ export class ApiError extends Error {
      *
      * @type {string|null}
      */
-    this.errorMessage = data.error_message || null;
+
```

Comment:
Can you remove the blank line that was added here between the JSDoc comment and the `this.errorMessage` assignment.